doc/source/index.rst

```diff
@@ -67,8 +67,9 @@ Documentation
 
        - :doc:`Quick start <tutorials/quickstart>`
        - :doc:`Gallery of examples <tutorials/index>`
+       - :doc:`Community Detection Guide <community_detection_guide>`
```

Comment:
Let's capitalize the first word only to be consistent with the other headings in this section. (I know it's a matter of style, I'm just suggesting this for sake of consistency)